main.c

```diff
@@ -116,6 +117,8 @@ int main(int argc,char **argv) {
             "            (helps in some rare cases)\n"
             "            Setting to 1.1.1.2 uses outgoing interface address and source port.\n"
             "            (helps in some rare cases)\n"
+            "    -t      sets the destination IP of forwarded packets; otherwise the\n"
+            "            original target is used\n"
```

Comment:
It might be worth mentioning the special `255.255.255.255` address here, as well, as this will be what a (potential) user will see when the ask for help.

Reply:
Ok.  I think I have done so now.  I also added a period at the end of the first sentence (realized I had previously missed it before where all the other arg descriptions had one).